test/components/views/dialogs/InviteDialog-test.tsx

```diff
@@ -291,13 +303,16 @@ describe("InviteDialog", () => {
     });
 
     it("should allow to invite multiple emails to a room", async () => {
-        render(<InviteDialog kind={InviteKind.Invite} roomId={roomId} onFinished={jest.fn()} />);
+        renderDialog();
 
         await enterIntoSearchField(aliceEmail);
         expectPill(aliceEmail);
 
         await enterIntoSearchField(bobEmail);
         expectPill(bobEmail);
+
+        // Take a snapshot of the input area inside address bar where those two pills are rendered
+        expect(document.querySelector(".mx_InviteDialog_editor")).toMatchSnapshot();
```

Comment:
If we already assert that the pills are present, what is the point of the snapshot?

Reply:
To save the DOM structure: https://github.com/matrix-org/matrix-react-sdk/pull/10791/files#diff-12257ca8f760d8a8714401e07a59367b30f61b58a0e5a82f7bea1ecffc16a4b3R3-R77

Is this redundant?